cluster/addons/cluster-monitoring/influxdb/influxdb-grafana-petset.yaml

```diff
@@ -65,8 +65,12 @@ spec:
             mountPath: /var
       volumes:
       - name: influxdb-persistent-storage
+{% if grains['cloud'] == 'gce' %}
```

Comment:
Bummer. We are really trying to avoid sticking jinja templates in the yaml files and instead creating "sed-friendly" files as we did for DNS (https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/dns). 

Also, if you do have to template this file, then it needs to be done in the same way as influxdb-pv.yaml where all of the conditionals are at the top of the file (and easy to remove with sed) and only basic substitution is done in the body of the file.